tests/forms/FormFieldTest.php

```diff
@@ -3,7 +3,52 @@
  * @package framework
  * @subpackage tests
  */
-class FormFieldTest extends SapphireTest implements TestOnly {
+class FormFieldTest extends SapphireTest {
+
+	public function testDefaultClasses() {
+		Config::nest();
+
+		Config::inst()->update('FormField', 'default_classes', array(
+			'class1',
+		));
+
+		$field = new FormField('MyField');
+
+		$this->assertContains('class1', $field->extraClass(), 'Default class not added to class list');
+
```

Comment:
Aren't they normally written in the positive sense? https://github.com/silverstripe/silverstripe-framework/blob/3.1/tests/model/DataObjectTest.php#L1205